docroot/profiles/custom/cgov_site/modules/custom/cgov_list/config/install/core.entity_view_display.paragraph.cgov_media_link.ncids_list_item_title_desc_image.yml

```diff
@@ -0,0 +1,55 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - core.entity_view_mode.paragraph.ncids_list_item_title_desc_image
+    - field.field.paragraph.cgov_media_link.field_media_link
+    - field.field.paragraph.cgov_media_link.field_override_image_promotional
+    - field.field.paragraph.cgov_media_link.field_override_list_description
+    - field.field.paragraph.cgov_media_link.field_override_title
+    - paragraphs.paragraphs_type.cgov_media_link
+  module:
+    - layout_builder
+third_party_settings:
+  layout_builder:
+    enabled: false
+    allow_custom: false
+id: paragraph.cgov_media_link.ncids_list_item_title_desc_image
+targetEntityType: paragraph
+bundle: cgov_media_link
+mode: ncids_list_item_title_desc_image
+content:
+  field_media_link:
+    type: entity_reference_entity_view
+    label: above
+    settings:
+      view_mode: ncids_collections_media
```

Comment:
This should be using the view mode of the image you put in place for nodes. (Which yes, means you need a view mode for media with the same name)

You then render the link as that view mode and it picks up the promo image field, and renders **that** promo image field as a `ncids_collections_media`.

Reply:
The current cgov_lists are using the paragraph view mode (list_item_title_desc & list_item_title_desc_image). Also when looking on the admin interface, you can only select paragraph view modes for media links.